source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBaseCardElementRenderer.mm

```diff
@@ -17,9 +17,9 @@ + (ACRCardElementType)elemType
 /* not implemented ATM */
 - (void)setSeparationConfig:(UIView *)viewGroup
 {
-    //SeparationStyle separationStyle,
-    //SeparationConfig defaultSeparationConfig,
-    //SeparationConfig strongSeparationConfig,
+    // SeparationStyle separationStyle,
+    // SeparationConfig defaultSeparationConfig,
+    // SeparationConfig strongSeparationConfig,
     //(Boolean) horizontalLine)
```

Comment:
this looks like this should be filed as a bug

Reply:
I don't know why it's not `// (Boolean) horizontalLine)`, and I didn't see a rule to make it remove the space.
I assume it's a bug of clang-format itself, and I would just ignore it.